tests/unit_tests/data_steward/tools/load_vocab_test.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+
+import mock
+from google.cloud.bigquery import Dataset, DatasetReference, Client
+from google.cloud.storage import Blob
+
+import common
+from tools import load_vocab
+
+
+class LoadVocabTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        project_id = 'fake_project_id'
+        dataset_id = 'fake_dataset_id'
+        bucket_name = 'fake_bucket'
+        dataset_ref = DatasetReference(project_id, dataset_id)
+        self.dst_dataset = Dataset(dataset_ref)
+        self.bq_client = Client(project_id)
+        self.bucket_name = bucket_name
+
+    def test_load_stage(self):
```

Comment:
Unit test failed on CircleCI and also failed locally for me

Reply:
Looks like this was never fully implemented. I refactored the test so it checks that expected calls to load_table_from_uri are made. Let me know if it looks ok.